.gitignore

```diff
@@ -0,0 +1 @@
+test.md*
```

Comment:
- [ ] The asterix (`*`) after the "test.md" is not necessary that will prevent any files that start with test.md we only want to ignore the test.md files not files that start with test.md